source/draw/DrawState.ooc

```diff
@@ -38,6 +38,8 @@ DrawState: cover {
 	_sourceNormalized := FloatBox2D new(0.0f, 0.0f, 1.0f, 1.0f)
 	_focalLengthNormalized := 0.0f // Relative to image width
 	targetTransform := FloatTransform2D identity // 2D scaling done after projection matrix but before w division
+	nearPlane: Float = 0.01f
+	farPlane: Float = 12500.0f
```

Comment:
Yeah but I was thinking more in term of that `_createProjection` could be called w/o those parameters then. Never mind though, just a thought.